backend/app/models/model_evaluation.py

```
@@ -0,0 +1,107 @@
+        foreign_key="project.id", nullable=False, ondelete="CASCADE"
+    )
+    organization_id: int = Field(
+        foreign_key="organization.id", nullable=False, ondelete="CASCADE"
+    )
+    is_deleted: bool = Field(default=False, nullable=False)
+
+    inserted_at: datetime = Field(default_factory=now, nullable=False)
+    updated_at: datetime = Field(default_factory=now, nullable=False)
+    deleted_at: datetime | None = Field(default=None, nullable=True)
+
+    project: "Project" = Relationship()
```

Ruff (0.12.2)

Undefined name `Project`

(F821)